### ISSUE TITLE
Switch  swiftmailer/swiftmailer to symfony/mailer

### ISSUE THREAD

**Issue by tunsco2010** [nWidart/laravel-modules#1324]
Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.

**Comment by dcblogdev** [comment]
I think this is a core Laravel issue and will be taken care of in a future release of Laravel.

**Comment by dcblogdev** [comment]
Yes, The Emailer is moving from SwiftMailer to Symfony Mailer this is happening in Laravel 9.
